UI/Pages/Shared/_OptionsSelect.cshtml

```diff
@@ -14,7 +14,7 @@
 	<div id="list-of-@Model.Name" role="group" aria-labelledby="option-select-title-@Model.Name" class="app-c-option-select__container js-options-container" tabindex="-1" >
 		<div class="app-c-option-select__container-inner">
 			<div class="gem-c-checkboxes govuk-form-group govuk-checkboxes--small" data-module="gem-checkboxes">
-				<fieldset class="govuk-fieldset govuk-checkboxes--small govuk-!-margin-0">
+				<fieldset class="govuk-fieldset govuk-checkboxes--small govuk-!-margin-1">
```

Comment:
So simple!

It still looks not quite centred.  Does `margin-2` make it fully in the centre?

Reply:
@benagain  You Are right made margin-2 an added the screenshot looks good now